src/Pages/Rules-Page/QuizRule.jsx

```diff
@@ -1,11 +1,13 @@
 import "./quizrule.css";
 import { useNavigate } from "react-router-dom";
+import { useQuiz } from "../../Context/QuizContext";
 export const QuizRule = () => {
   const navigate = useNavigate();
+  const { state } = useQuiz();
```

Comment:
this is fine - just a suggestion you can destructure ```currentQuiz``` like: ```{ state: {currentQuiz} }```

and there you have to add ```currentQuiz.category```

Reply:
Okay, I will update in the next PR.
ThankYou